README.md

```diff
@@ -139,7 +124,7 @@ To use with Meson, you just need to add `dependency('pistache')` as a dependency
 project(
     'MyPistacheProject',
     'cpp',
-    meson_version: '>=0.57.0'
+    meson_version: '>=0.55.0'
 )
```

Comment:
When I run `./debian/rules get-orig-source` to produce the upstream tarball, I see the following:

```
WARNING: Project targeting '>=0.50.0' but tried to use feature introduced in '0.54.0': meson.override_dependency.
Run-time dependency libcurl found: YES 7.68.0
Run-time dependency GTest found: YES 1.10.0
Program valgrind found: YES
Program gcov found: YES
Program cppcheck found: YES
Run-time dependency rapidjson found: YES 1.1.0
Build targets in project: 35
WARNING: Project specifies a minimum meson_version '>=0.50.0' but uses features which were added in newer versions:
 * 0.54.0: {'meson.override_dependency'}
```

Reply:
Strange. If you look at [src/meson.build#L53](https://github.com/Tachi107/pistache/blob/meson/src/meson.build#L53), I explicitly check the Meson version, so it is a false positive.